tests/Company/Serializers/CompanySerializerTest.php

```diff
@@ -0,0 +1,70 @@
+<?php declare(strict_types=1);
+
+namespace VSV\GVQ_API\Company\Serializers;
+
+use PHPUnit\Framework\TestCase;
+use VSV\GVQ_API\Company\Models\Company;
+use VSV\GVQ_API\Factory\ModelsFactory;
+use VSV\GVQ_API\Factory\ExpectedJsonTrait;
+
+class CompanySerializerTest extends TestCase
+{
+    use ExpectedJsonTrait;
+
+    /**
+     * @var CompanySerializer
+     */
+    private $serializer;
+
+    /**
+     * @var string
+     */
+    private $companyAsJson;
+
+    /**
+     * @var Company
+     */
+    private $company;
+
+    protected function setUp(): void
+    {
+        $this->serializer = new CompanySerializer();
+
+        $this->companyAsJson = $this->getExpectedJson(__DIR__.'/../../Factory/Samples/company.json');
```

Comment:
Nu we een factory hebben om de models aan te maken hebben we deze trait niet meer nodig. We kunnen de methode verhuizen naar de factory een de naam aanpassen naar `createJson`. Dit mag je ook al doen voor de bestaande testen in Question.